ajax/libs/codemirror-spell-checker/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "codemirror-spell-checker",
+  "version": "1.1.2",
+  "filename": "spell-checker.min.js",
+  "description": "Dead-simple spell checking in CodeMirror",
+  "homepage": "http://nextstepwebs.github.io/codemirror-spell-checker/",
+  "keywords": [
+    "codemirror",
+    "code",
+    "mirror",
+    "spell",
+    "checker",
+    "checking",
+    "spellchecker",
+    "spellchecking",
+    "javascript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/NextStepWebs/codemirror-spell-checker"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Wes Cossick",
+    "url": "http://www.WesCossick.com"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/NextStepWebs/codemirror-spell-checker.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*.!(*aff|*dic)"
+        ]
+      }
```

Comment:
@extend1994 We don't really need files from `src` since they are all build files and cannot be used directly by a developer on their website through cdnjs. All we need are the files in `dist` 😄